client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -28,11 +28,26 @@
 import scala.runtime.AbstractFunction1;
 
 import org.apache.uniffle.client.util.RssClientConfig;
+import org.apache.uniffle.common.config.ConfigOption;
+import org.apache.uniffle.common.config.ConfigOptions;
 import org.apache.uniffle.common.config.ConfigUtils;
 import org.apache.uniffle.common.config.RssConf;
 
 public class RssSparkConfig {
 
+  public static final ConfigOption<Long> RSS_CLIENT_SEND_SIZE_LIMITATION = ConfigOptions
```

Comment:
This is an existing config, I just make it as rssConf style.

Reply:
> Ah, I see. But why this configuration is in RssSparkConf, rather in a RssBaseConf?

rather in a `RssClientConf` ? If conf in `RssClientConf`, it will be shareable for MR/spark. If only in `RssSparkConf`, it will only be visible for Spark. Of course, this conf is only for spark.

Reply:
Yes, it's OK. But I hope it could retrieved by RssConf in WriteBufferManager(This class don't have sparkConf var)

Reply:
Oh... I have removed the original `ConfigEntry<String> RSS_CLIENT_SEND_SIZE_LIMIT = createStringBuilderConfigEntry<String> RSS_CLIENT_SEND_SIZE_LIMIT = createStringBuilder`. Please check it.